src/js/controller/model.ts

```diff
@@ -288,7 +288,7 @@ class Model extends SimpleModel {
     // Desktop players must have autostart set to viewable
     setAutoStart(autoStart?: AutoStart): void {
         // Always autostart for tizen app
-        if (OS.tizen) {
```

Comment:
This has nothing to do with controls. The idea is Tizen does not support the autostart check, and never blocks autostart, regardless of whether it's browser or app.

Reply:
I thought this check was the one that resolved autoplay for tizen
https://github.com/jwplayer/jwplayer/blob/745778d7eac6fe7163d6d8bd35871d821c28e941/src/js/utils/can-autoplay.js#L54-L57
in https://github.com/jwplayer/jwplayer/pull/3626

whereas this check is always setting autostart to true so that the videos will autoplay for the tizen app

Reply:
OK. That sounds like the app's responsibility though (config), not the player.

Reply:
If you do this here it also limits the flexibility of the API to be able to setup and create players that only play on command.

Reply:
Moved this change to the app side: https://github.com/jwplayer/jwplayer-tizen-app/pull/10